HACKING

```diff
@@ -15,30 +15,31 @@ Join #elementary-dev on Freenode (irc.freenode.net).
 Go through problem reports and check unconfirmed bugs or those lacking
 information and mark any duplicates you spot.
 
-    http://bugs.launchpad.net/gala
+    https://github.com/elementary/gala/issues
 
 Help getting Gala translated in your language!
 
-    https://translations.launchpad.net/gala
+    https://l10n.elementary.io/projects/desktop/gala/
 
 Answer questions.
 
-    https://answers.launchpad.net/gala
+    https://elementaryos.stackexchange.com/questions/tagged/gala
 
 ====== Check out the sources ======
 
-    bzr branch lp:gala
+    git clone https://github.com/elementary/gala.git
 
 The development trunk (master, tip) is the latest iteration of the next
 release. Browse it online and look for other branches at:
 
-    http://code.launchpad.net/gala
+    https://github.com/elementary/gala
 
 ====== Build the code ======
 
 Prepare the source and compile:
-    ./autogen.sh --prefix=/usr
+    ./autogen.sh --prefix=/usr --libdir=/usr/lib/x86_64-linux-gnu/
```

Comment:
--libdir=/usr/lib/x86_64-linux-gnu/ is an architecture and system dependent command, better omit it here